test/integration_test.py

```diff
@@ -709,3 +719,59 @@ def test_mini_dss_failures(self):
                 mini_dss.get_file(uuid, version, 'aws')
             with self.assertRaises(self.SpecialError):
                 mini_dss.get_native_file_url(uuid, version, 'aws')
+
+
+class AzulChaliceLocalIntegrationTest(AzulTestCase):
+    service_app = Path(config.project_root) / 'lambdas' / 'service'
+    host = '0.0.0.0'
+    port = 8000
+    local_chalice_server = None
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+        localfactory = factory.CLIFactory(str(cls.service_app))
+        config = localfactory.create_config_obj()
+        cls.local_chalice_server = localfactory.create_local_server(app_obj=app,
+                                                                    config=config,
+                                                                    host=cls.host,
+                                                                    port=cls.port)
+        thread = threading.Thread(target=cls.local_chalice_server.serve_forever)
+        thread.start()
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        cls.local_chalice_server.shutdown()
+        super().tearDownClass()
+
+    def test_local_chalice_health_endpoint(self):
+        base_url = furl(scheme='http',
+                        host=self.host,
+                        port=self.port,
+                        path='health')
+        response = requests.get(base_url.url)
+        self.assertEqual(response.status_code, 200)
+
+    paths = ('summary', 'bundles', 'files', 'projects', 'samples')
+
+    def test_local_chalice_index_endpoints(self):
+        for path in self.paths:
+            with self.subTest(path=path):
+                base_url = furl(scheme='http',
+                                host=self.host,
+                                port=self.port,
+                                path=f'index/{path}')
+                response = requests.get(base_url.url)
+                self.assertEqual(response.status_code, 200)
+
+    def test_local_filtered_index_endpoints(self):
+        filters = {"genusSpecies": {"is": ["Homo sapiens"]}}
+        for path in self.paths:
+            with self.subTest(path=path):
+                base_url = furl(scheme='http',
+                                host=self.host,
+                                port=self.port,
+                                path=f'index/{path}',
+                                query=f'filters={str(filters)}')
```

Comment:
This test is failing for me locally. I think it's because you're casting filters to a string instead of calling `json.dumps`